### PR TITLE
Feature/mtm 56466 dashboards redesign annoucement

### PR DESCRIPTION
**Pull request by DawidSAG**

### CODE REVIEW

**Review by janhommes — APPROVED** [review]

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**Functional Improvements**:
+
+* We have introduced an edit mode for dashboards.
+* The lock/unlock feature has been replaced with an edit mode for better control and clarity.
+
+**New Features**:
+
+* Versioning to manage and track dashboard changes.
+* Undo/redo functionality for flexible editing.
```
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:31]:
```suggestion
* Possibility to enable and disable dashboard template for device dashboards.
* Versioning to manage and track dashboard changes.
* Undo/redo functionality for flexible editing.
```

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md]: done

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56466
+version: 1020.0.0
+---
+In a future version, we are introducing redesigned dashboards with both visual and functional enhancements, as well as new functionalities.
+
+**Visual and UX Enhancements**:
```
Comment (DawidSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:19]:
```suggestion
**Visual and UX enhancements**:
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56466
+version: 1020.0.0
+---
+In a future version, we are introducing redesigned dashboards with both visual and functional enhancements, as well as new functionalities.
```
Comment (DawidSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:17]:
```suggestion
In a future version, we will introduce redesigned dashboards with both visual and functional enhancements, as well as new functionalities.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+**Visual and UX Enhancements**:
+
+* We have redesigned the way dashboards are edited.
+
+**Functional Improvements**:
```
Comment (DawidSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:23]:
```suggestion
**Functional improvements**:
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+**Visual and UX enhancements**:
+
+* We have redesigned the way dashboards are edited.
+
+**Functional improvements**:
+
+* We have introduced an edit mode for dashboards.
+* The lock/unlock feature has been replaced with an edit mode for better control and clarity.
+
+**New Features**:
```
Comment (DawidSAG) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:28]:
```suggestion
**New features**:
```

**Review by jdreSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56466
+version: 1020.0.0
+---
+In a future version, we will introduce redesigned dashboards with both visual and functional enhancements, as well as new functionalities.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:17]:
```suggestion
In a future version, we will introduce redesigned dashboards with both visual and functional enhancements as well as new functionalities.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+---
+In a future version, we will introduce redesigned dashboards with both visual and functional enhancements as well as new functionalities.
+
+**Visual and UX enhancements**:
+
+* We have redesigned the way dashboards are edited.
+
+**Functional improvements**:
+
+* We have introduced an edit mode for dashboards.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:25]: This is included in the second item, isn´t it?

**Review by rahultsag — CHANGES_REQUESTED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-56466
+version: 1020.0.0
+---
+In a future version, we will introduce redesigned dashboards with both visual and functional enhancements as well as new functionalities.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:17]:
```suggestion
In the upcoming version, we will introduce redesigned dashboards with both visual and functional enhancements as well as new functionalities.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+---
+In the upcoming version, we will introduce redesigned dashboards with both visual and functional enhancements as well as new functionalities.
+
+**Visual and UX enhancements**:
+
+* We have redesigned the way dashboards are edited.
+
+**Functional improvements**:
+
+* We have introduced an edit mode for dashboards.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:25]:
```suggestion
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+**Visual and UX enhancements**:
+
+* We have redesigned the way dashboards are edited.
+
+**Functional improvements**:
+
+* The lock/unlock feature has been replaced with an edit mode for better control and clarity.
+
+**New features**:
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:27]:
```suggestion
**New functionalities**:
```

**Review by rahultsag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@DawidSAG Could we add a link to the user documentation such as "For details refer to xxx".

**Comment by DawidSAG** [comment]
> @DawidSAG Could we add a link to the user documentation such as "For details refer to xxx".

You would like to add link to: https://cumulocity.com/docs/cockpit/working-with-dashboards/ ?

**Comment by BeateRixen** [comment]
> > @DawidSAG Could we add a link to the user documentation such as "For details refer to xxx".
> 
> You would like to add link to: https://cumulocity.com/docs/cockpit/working-with-dashboards/ ?

Yes, I think it makes sense to point users to the place where they can find more details.

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+* The lock/unlock feature has been replaced with an edit mode for better control and clarity.
+
+**New functionalities**:
+
+* Possibility to enable and disable dashboard template for device dashboards.
+* Versioning to manage and track dashboard changes.
+* Edit mode with undo/redo functionality for flexible adjustments.
+
+For details refer to: [Working with dashboards](https://cumulocity.com/docs/cockpit/working-with-dashboards/).
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-dashboards-redesign-and-improvements.md:33]:
```suggestion
For details refer to [Working with dashboards](https://cumulocity.com/docs/cockpit/working-with-dashboards/).
```

**Pull request merged**